doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,117 @@
+* If necessary, you configured Spring security for the application. For instructions about configuring Spring security, see xref:bus-app-security-con_{context}[].
+* You completed other Spring configuration for the business application, as necessary. For instructions about Spring configuration for your business application, see xref:bus-app-configure-con_{context}[].
+* You created a single JAR file for the business application. For instructions about creating a single JAR file for your SpringBoot business application, see xref:creating-self-contained-image-proc_{context}[].
+* You are logged on to your {OPENSHIFT} environment using the `oc` command and the required project is active.
+ifdef::PAM[]
+* If you want to use {CENTRAL} Monitoring, you installed {CENTRAL} Monitoring using the operator. For instructions about installing {CENTRAL} Monitoring and other {PRODUCT} components using the operator, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_]. For convenience, use the same project/namespace to deploy {CENTRAL} Monitoring and your business application.
++
+[IMPORTANT]
+====
+You must configure {CENTRAL} Monitoring to use the controller startup strategy. To enable the controller strategy on {CENTRAL} Monitoring, in the *Console* tab of the operator configuration, add a `KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED` environment variable and set it to `false`.
+====
+endif::PAM[]
+
+.Procedure
+
+. Outside the business application project directories, create an `ocp-image` directory with the following subdirectories:
++
+----
+ocp-image
+|--/root
+  |--/opt
+    |-- /spring-service
+----
++
+. Copy the single JAR file for your business application into the `root/opt/spring-service` subdirectory, for example:
++
+----
+cd ../business-application-service
+cp target/business-application-service-1.0-SNAPSHOT.jar ../ocp-image/root/opt/spring-service/
+----
++
+. In the `ocp-image` directory, create a `Dockerfile` file with the following content:
++
+----
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
+COPY root /
+EXPOSE 8090
+WORKDIR /opt/spring-service/
+CMD ["sh","-c", "java ${JAVA_OPTIONS} -Dorg.kie.server.mode=PRODUCTION -jar /opt/spring-service/<FILENAME>.jar"]
+----
++
+Replace `<FILENAME>.jar` with the name of the single JAR file for your business application.
++
+ifdef::PAM[]
+Alternatively, if you want to configure communication with {CENTRAL} Monitoring, use the following content for `Dockerfile`:
++
+----
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
+COPY root /
+EXPOSE 8090
+WORKDIR /opt/spring-service/
+CMD ["sh","-c", "java ${JAVA_OPTIONS} -Dorg.kie.server.location=${KIESERVER_LOCATION} -Dorg.kie.server.controller=${KIESERVER_CONTROLLERS} -Dorg.kie.server.controller.user=${KIE_SERVER_CONTROLLER_USER} -Dorg.kie.server.controller.pwd=${KIE_SERVER_CONTROLLER_PWD} -Dorg.kie.server.mode=PRODUCTION -jar /opt/spring-service/<FILENAME>.jar"]
```

Comment:
I am not sure about this one (org.kie.server.location).
Shouldn't it be part of the application.properties ?
https://github.com/kiegroup/droolsjbpm-integration/blob/c44df5fd24fc3c172d2b04b777b198f73b783143/kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration/src/main/java/org/kie/server/springboot/autoconfiguration/KieServerProperties.java#L34

https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration/src/main/java/org/kie/server/springboot/autoconfiguration/KieServerAutoConfiguration.java#L111

It seems autoconfiguration will set that system property based on the value in app.properties.